.travis.yml

```diff
@@ -5,25 +5,27 @@ cache:
     - $HOME/.m2/repository/
     - $HOME/.ivy2/cache/
     - $HOME/.sbt/boot/
+    - $HOME/.coursier
 
 sudo: false
 
-jdk:
-  - openjdk7
-  - oraclejdk8
-
 branches:
   only:
     - develop
 
-script:
-  - ./sbt jcheckStyle
-  - ./sbt test
-  - ./sbt test -J-Dmsgpack.universal-buffer=true
-
 matrix:
   include:
-  - dist: trusty
+  - env: PROJECT=checkstyle
+    jdk: oraclejdk8
+    script:
+    - ./sbt jcheckStyle
+  - env: PROJECT=java8
+    jdk: oraclejdk8
+    script:
+    - ./sbt test
```

Comment:
add `sbt ";scalafmt::test ;test:scalafmt::test ;sbt:scalafmt::test"` ?

https://github.com/lucidsoftware/neo-sbt-scalafmt/blob/1.14/README.md#usage
https://github.com/sbt/sbt/blob/v1.0.4/.travis.yml#L20


> To ensure everything is formatted, and fail if it is not (e.g. as a CI step),
>
> `>` scalafmt::test      # check compile sources
> `>` test:scalafmt::test # check test sources
> `>` sbt:scalafmt::test  # check .sbt sources

Reply:
Ah, Maybe I was wrong. You added `addSbtPlugin("com.geirsson" % "sbt-scalafmt"    % "1.3.0")`, not neo-sbt-scalafmt 🙇

Reply:
Right. sbt-scalafmt reformats everything including main, test, *.sbt, etc.